packages/react/src/tooltip/OverflowTooltip.js

```diff
@@ -7,6 +7,10 @@ import Tooltip from './Tooltip';
 const OverflowTooltip = forwardRef((
   {
     children,
+    disabled: disabledProp,
+    nextToCursor: nextToCursorProp,
+    offset: offsetProp = [0, 8],
+    placement: placementProp = 'bottom',
     ...rest
```

Comment:
Apply the same default value in v2 branch:
https://github.com/trendmicro-frontend/tonic-ui/blob/v2/packages/react/src/tooltip/OverflowTooltip.js

```is
const OverflowTooltip = forwardRef((
  {
    children,
    disabled: disabledProp = false,
    nextToCursor: nextToCursorProp = true,
    offset: offsetProp = [8, 12],
    placement: placementProp = 'bottom-end',
    ...rest
  },
  ref,
) => {
```

Reply:
updated in https://github.com/trendmicro-frontend/tonic-ui/pull/925/commits/5942f0a8d3fd179bde6a89b6ce587370a81cf134